XLForm/XL/Descriptors/XLFormDescriptor.h

```diff
@@ -73,7 +73,7 @@ typedef NS_ENUM(NSInteger, XLFormErrorCode)
 
 -(NSIndexPath *)indexPathOfFormRow:(XLFormRowDescriptor *)formRow;
 
--(NSDictionary *)formValues;
+@property (nonatomic, readonly, copy) NSDictionary *formValues;
```

Comment:
@jkereako I would like to keep `formValues` as a method since is not a property.